website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1748,7 +1748,7 @@ $ curl \
 
 This endpoint force unlocks a workspace. Only users with admin access are authorized to force unlock a workspace.
 
-For a force unlock, Terraform Cloud attempts to find the latest finalized provisional state version and sets it as the current state version for a workspace. Terraform Cloud may select an earlier provisional state version as the current version for a workspace. This only occurs if Terraform Cloud has not yet finalized the latest provisional state version. Terraform Cloud will not prevent a force unlock if no finalized provisional state versions are available.
+For a force unlock, Terraform Cloud attempts to find the latest finalized intermediate state version and sets it as the current state version for a workspace. Terraform Cloud may select an earlier intermediate state version as the current version for a workspace. This only occurs if Terraform Cloud has not yet finalized the latest intermediate state version. Terraform Cloud will not prevent a force unlock if no finalized intermediate state versions are available.
```

Comment:
Let me rewrite this because I fixed this behavior!!

```suggestion
When unlocking a workspace, Terraform Cloud sets the newest uploaded state version as the current state version for the workspace. During unlock, pending state versions that are created but not yet uploaded become discarded. The Terraform CLI ensures that state is both created and uploaded successfully when attempting to save it.
```